pkg/cmd/list.go

```diff
@@ -49,6 +49,11 @@ NONRESOURCEURL is a partial URL that starts with "/".`
 
   # List who can access the URL /logs/
   kubectl who-can get /logs`
+
+	// RoleKind is the RoleRef's Kind referencing a Role.
+	RoleKind = "Role"
```

Comment:
I'm OK with this as it's unlikely to change, but is there not a definition we could pull in from kubernetes somewhere?

Reply:
Unfortunately not. I did search in project's dependencies and it seems that it's common to repeatedly type in `"Role"` and `"CusterRole"` literals all over the place ;)